site/docs/src/json/users/forgot-password-request-advanced.json

```diff
@@ -1,5 +1,8 @@
 {
   "changePasswordId": "YkQY5Gsyo4RlfmDciBGRmvfj3RmatUqrbjoIZ19fmw4",
   "loginId": "example@fusionauth.io",
-  "sendForgotPasswordEmail": false
-}
\ No newline at end of file
+  "sendForgotPasswordEmail": false,
+  "state" : { 
+    "lastPage" : "/buy/123"
```

Comment:
I don't know how the rest of the doc is, but should it be 
  
`    "lastPage" : "/foo/baz"`
    
  
 or is that too generic?

Reply:
Nah, good suggestion.